### DISCUSSION TITLE
Next.js separate import styles

### DISCUSSION THREAD

**Discussion by Robert126** [gpbl/react-day-picker#1574] [Support] [answered]
How can I use react-day-picker without the custom styling?
And download the styling css files separately?

Next.js does not allow components from node_modules to import styles. You'll need custom styling.

https://github.com/wojtekmaj/react-date-picker#custom-styling

Thanks for your help!
Kind regards,
Robert

**Comment by gpbl** [comment, marked as answer]
@Robert126 you can copy and import the [style.css](https://github.com/gpbl/react-day-picker/blob/master/packages/react-day-picker/src/style.css) into your app, instead of importing it from node modules (e.g. `react-day-picker/dist/style.css`).
